learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -16,7 +16,7 @@ Please be advised that many of the search products described below are constantl
 
 ## Comparisons
 
-### MeiliSearch vs. Elasticsearch
+### MeiliSearch vs elasticsearch
```

Comment:
Elasticsearch is a proper noun (we are speaking about a company called Elasticsearch).

Suggestion: `MeiliSearch vs Elasticsearch`